### PR TITLE
Support multiple environments for feature toggles

### PR DESCRIPTION
**Pull request by jackrust**
Extends angular feature flags to support /beta and live environments
supports query parameter ("?flags") to display feature flags
@PageUpPeopleOrg/redfins

### CODE REVIEW

File: demo/data/flags.json [hunk 1]
```diff
@@ -1,26 +1,45 @@
-[
-    {
+{
+    "beta": [{
```
Comment (phdesign) [inline demo/data/flags.json]: What was the thinking behind this? Are we creating a lot of work for ourselves maintaining two collections of flags? And there's only one code base right so the feature flag needs to either exist in the code base or not, regardless of which environment we're running on.

File: dist/featureFlags.js [hunk 2]
```diff
@@ -216,4 +227,36 @@ angular.module('feature-flags').provider('featureFlags', function() {
     }];
 });
 
+angular.module('release-toggle', [])
```
Comment (phdesign) [inline dist/featureFlags.js]: Can we give this a different name? release-toggle is just another way of saying feature-flags. Is this the UI for it? Maybe it should be feature-flag-dialog? Or feature-flag-ui?

**Comment by phdesign** [comment]
Adding `?flags` to the demo doesn't seem to show the flags dialog?
I expected this to work. `http://localhost:9999/beta/?flags`

File: demo/data/flags.json (hunk 1 above)
Comment (KylieTims) [inline demo/data/flags.json]: I guess the thinking was that we need to be able to turn a flag both on and off in both beta and live environments. This is what we thought would be the easiest way of doing it with minimal changes required to the existing code.

File: dist/featureFlags.js (hunk 2 above)
Comment (KylieTims) [inline dist/featureFlags.js]: Yep, good idea, I like featur-flag-ui (its not really a dialog its just a div on the page)

**Comment by KylieTims** [comment]
@phdesign - rename done, comments added

**Comment by phdesign** [comment]
Superceeded by a later PR

**Pull request closed without merging**